samples/cpp/stable_diffusion/CMakeLists.txt

```diff
@@ -26,3 +26,22 @@ install(TARGETS stable_diffusion
         RUNTIME DESTINATION samples_bin/
         COMPONENT samples_bin
         EXCLUDE_FROM_ALL)
+
+# create LoRA sample executable
+
+add_executable(lora_stable_diffusion
```

Comment:
typically, we have a folder per sample with its own readme,md file, which describes GenAI capabilities demonstrated in the sample

Reply:
What is the recommendation for other files that are shared between these samples: `imwrite.hpp/cpp`? Should we move them to a shared directory or duplicate in each sample directory making samples fully independent?

We are going to have adapters supported for each type of pipeline. There are two options for how we can organize the samples:

 - Create a dedicated sample for each combination LoRA + pipeline with its own directory; README will explain only LoRA-specific things explicitly referring to the "main" sample for the pipeline without LoRA. `lora_greedy_causal_lm` takes this path but doesn't have README yet.
 - Add `lora.cpp` file in each relevant existing sample; add LoRA section to the main README file for the sample. Old SD samples demonstrated that approach, and this PR gives an example how it looks but still README is not ready.

I don't see much difference between them, let's align and pick one.

Reply:
I'm Ok to have a folder per use case:
- text generation / llm
- text to image
- whisper
- visual language 

with multiple samples within a folder.
Each folder will have a common readme with generic use-case description and small per-sample section (e.g. what sample demonstrates, commands to get models and run app)